scripts/start_parallel_jobs_slurm.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/packages/python/anaconda/2.1.0/bin/python
```

Comment:
It was something I did in order to test our local version, which I should have reverted back to original. Sorry about that! I'll fix that.